src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -195,7 +201,14 @@ export class SVGViewer extends React.Component<SvgViewerProps, SvgViewerState> {
             searchClassName={customClassNames.searchResults}
             currentSearchClassName={customClassNames.currentSearchResult}
           />
-          <div ref={this.pinchZoomContainer}>
+          <div
+            ref={this.pinchZoomContainer}
+            style={
+              !isDesktop && this.state.isSearchVisible
```

Comment:
we use `&&` and `||` in line 144, is that expected?

Reply:
yes, I left comments for it